internal/repository/prisma/event.go

```diff
@@ -79,6 +80,15 @@ func (r *eventRepository) ListEvents(tenantId string, opts *repository.ListEvent
 		countParams.Keys = opts.Keys
 	}
 
+	if opts.Workflows != nil {
+		var wfs []pgtype.UUID
+		for _, wf := range opts.Workflows {
+			wfs = append(wfs, sqlchelpers.UUIDFromStr(wf))
+		}
+		queryParams.Workflows = wfs
+		countParams.Workflows = wfs
```

Comment:
alternatively we can just do

```
queryParams.Workflows = opts.Workflows
```

but then we need to cast the `sqlc.narg('workflows')::uuid[]` to a string, not sure which one is uglier